blocks/header/header.css

```diff
@@ -0,0 +1,188 @@
+.header {
+  padding: 10px;
+  position: fixed;
+  width: 100%;
+  box-sizing: border-box;
+}
+
```

Comment:
With other parts that were added, while it's sticking to the top- it's going behind other content ( like the cards in services now) 
Should just need to add a z-index here to be good!